src/app/ApplicationTemplate.Shared.Views/Framework/Email/EmailService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Email;
+
+namespace ApplicationTemplate;
+
+public sealed class EmailService : IEmailService
+{
+	public async Task Compose(Email email)
+	{
+		var emailMessage = new EmailMessage()
+		{
+			Subject = email.Subject,
+			Body = email.Body,
+		};
+
+//-:cnd:noEmit
+/* TODO: Use Uno implementation for Android & iOS when it's ready.
```

Comment:
Sending logs by email is a mandatory feature. If Uno doesn't support it we need an alternative.

CC @MatFillion

Reply:
I could use parts of the Essentials implementation.